src/operator/nn/activation-inl.h

```diff
@@ -199,13 +199,8 @@ void ActivationGradCompute(const nnvm::NodeAttrs& attrs,
                            const std::vector<OpReqType>& req,
                            const std::vector<TBlob>& outputs) {
   const ActivationParam& param = nnvm::get<ActivationParam>(attrs.parsed);
-#if (MXNET_USE_CUDNN == 1 || MXNET_USE_MKLDNN == 1)
-  bool relu = param.act_type == activation::kReLU;
+  const bool relu = param.act_type == activation::kReLU;
   CHECK_EQ(inputs.size(), relu ? 2U : 3U);
```

Comment:
This code was added in https://github.com/apache/incubator-mxnet/pull/11827 . Maybe @samskalicky can help here and say whether it's needed

Reply:
I fixed all this stuff in https://github.com/apache/incubator-mxnet/pull/13409  just keeping the CI pipeline here until @marcoabreu  green lights.